runtime/src/test/resources/tailcall/runtime/graphql/execution/test-resolve-with-env.graphql

```diff
@@ -0,0 +1,15 @@
+#> server-sdl
+schema {
+  query: Query
+}
+
+type Query {
+  identity: String @http(path: "/resolveWithEnv", query: {data: "{{env.foo}}"}  baseURL: "https://foo.com")
```

Comment:
```suggestion
  identity: String @http(path: "/resolveWithEnv", query: {data: "{{env.foo}}"}  baseURL: "https://foo.com/echo")
```

Reply:
add a echo route, that returns the the complete request as response. We should be able to assert on method, URL, body, query params etc. using this route.